_print_binary.c

```diff
@@ -0,0 +1,39 @@
+#include "main.h"
+
+/**
+ * _print_binary - prints an unsigned integer in binary format
+ * @args: A list of arguments containing the integer to print
+ *
+ * Return: number of characters printed (int)
+ */
+int _print_binary(va_list args)
+{
+    unsigned int num = va_arg(args, unsigned int);
+    int len = 0, j;
```

Comment:
`len` should be started at 1, because even if we got a null number `0`, the length will be equal to 1 because we printed 0

Reply:
Okay